ui/components/app/transaction-decoding/components/ui/accreditation/index.scss

```
@@ -0,0 +1,27 @@
+.accreditation {
+  display: flex;
+  font-size: 0.75rem;
+  display: flex;
+  margin-top: 10px;
+
+  &__tooltip {
+    padding-right: 5px;
+  }
+
+  &__info {
+    color: $ui-black;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__content {
+    padding: 0 4px;
+    text-transform: capitalize;
+
+    &--link.btn-link {
+      @include H7;
+
+      padding: 0;
+    }
+  }
+}
```

Since we are using the design system UI components we can simplify the styles. Also adhering to `8px` grid values(All margin and padding values should be divisible by 8. With the exception of `4px` 

```suggestion
.accreditation {
  display: flex;
  align-items: center;
  margin-top: 8px;

  &__icon {
    margin-right: 8px;
  }

  &__info {
    color: $ui-black;
    display: flex;
    flex-wrap: wrap;
  }

  &__link.btn-link {
    @include H7;
    padding: 0 4px;
    width: auto;
  }
}

```